### DISCUSSION TITLE
External Interaction Tagging

### DISCUSSION THREAD

**Discussion by x676f64** [projectdiscovery/nuclei-templates#7244] [Ideas]
I would like to propose adding a tag for plugins that have external interaction outside of interactsh enabled plugins.

As far as I'm aware and as per discussions in Discord, currently there is no way to assure a module of plugins will not make external interactions and there is no tag currently to indicate as much.

The reason for this is that there could be potentially information leaks or trigger firewall egress alerts.

**Comment by MetzinAround** [comment]
Moved to Ideas discussion. Thanks @x676f64
